pkg/systemd/superuser-alert.jsx

```diff
@@ -33,14 +33,27 @@ export class SuperuserAlert extends React.Component {
         super();
         this.superuser = cockpit.dbus(null, { bus: "internal" }).proxy("cockpit.Superuser", "/superuser");
         this.superuser.addEventListener("changed", () => { this.setState({}) });
+        this.state = { can_do_sudo: false };
+        cockpit.spawn(["sudo", "-v", "-n"], { err: "message" })
+                .fail(e => {
+                    if (e.message && e.message.match("Sorry, user .+ may not run sudo on .+\\.")) {
+                        this.setState({ can_do_sudo: false });
+                    } else
+                        this.setState({ can_do_sudo: true });
```

Comment:
This is confusing and needs a comment -- if `sudo -v -n` fails, why is it set to true? Also, same issues from above apply.